Source/WebGPU/WebGPU/UsdModelRenderer.swift

```diff
@@ -0,0 +1,428 @@
+import Foundation
+import Metal
+import QuartzCore
+import QuartzCore_Private
+import WebGPU_Internal
+#if canImport(DirectDrawBackend)
+import DirectDrawBackend
+import DirectResource
+import ShaderGraph
+#endif
+
+@objc
+@implementation
+extension DDBridgeVertexAttributeFormat {
+    let semantic: Int32
+    let format: Int32
+    let layoutIndex: Int32
+    let offset: Int32
+
+    init(
+        semantic: Int32,
+        format: Int32,
+        layoutIndex: Int32,
+        offset: Int32
+    ) {
+        self.semantic = semantic
+        self.format = format
+        self.layoutIndex = layoutIndex
+        self.offset = offset
+        super.init()
+    }
+}
+
+@objc
+@implementation
+extension DDBridgeVertexLayout {
+    let bufferIndex: Int32
+    let bufferOffset: Int32
+    let bufferStride: Int32
+
+    init(
+        bufferIndex: Int32,
+        bufferOffset: Int32,
+        bufferStride: Int32
+    ) {
+        self.bufferIndex = bufferIndex
+        self.bufferOffset = bufferOffset
+        self.bufferStride = bufferStride
+        super.init()
+    }
+}
+
+@objc
+@implementation
+extension DDBridgeAddMeshRequest {
+    var indexCapacity: Int32
+    var indexType: Int32
+    var vertexBufferCount: Int32
+    var vertexCapacity: Int32
+    var vertexAttributes: [DDBridgeVertexAttributeFormat]?
+    var vertexLayouts: [DDBridgeVertexLayout]?
+
+    init(value: Int32) {
```

Comment:
where is `value` used here?

Reply:
it's not used, but upon removing it I saw:
```
/Volumes/Data/OpenSource/Source/WebGPU/WebGPU/UsdModelRenderer.swift:61:5: error: overriding declaration requires an 'override' keyword
    init() {
/Volumes/Data/XcodeB.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX26.1.Internal.sdk/usr/include/objc/NSObject.h:66:1: note: overridden declaration is here
- (instancetype)init
^
```

because I had forgotten `override` on the Swift method. 

Adding `override` it seems to be fine with that change applied :)